### DISCUSSION TITLE
Disable "Create an account"

### DISCUSSION THREAD

**Discussion by calloatti** [exadel-inc/CompreFace#1080] [Q&A] [answered]
Is there a way to disable the "Create an account" link in the login page? I would like to disable it once I have created one account.

**Comment by pospielov** [comment, marked as answer]
There is no such possibility now. 
But new accounts don't have access to any applications and can't create new applications, so I don't see how it can be exploited.

**Reply by calloatti** [reply]
I see thank you, I did not test that, just assumed that any new account could do anything.
